cppcache/integration-test/testThinClientPartitionResolver.cpp

```diff
@@ -105,8 +105,8 @@ DUNIT_TASK_DEFINITION(CLIENT1, CreatePoolAndRegions)
     initClient(true);
 
     char endpoints[1024] = {0};
-    sprintf(endpoints, "%s,%s,%s", endpointNames.at(0), endpointNames.at(1),
-            endpointNames.at(2));
+    sprintf(endpoints, "%s,%s,%s", endpointNames.at(0).c_str(),
```

Comment:
My not just use `std::string` concatenation?

Reply:
I finally removed this code. It was useless.